docs/section19/19-20.md

```diff
@@ -0,0 +1,73 @@
+            - Get the hvac system total return fan kW, need to get the second element in the list of fan powers:`hvac_sys_total_return_fan_power_b = fan_sys_kW[1]` 
+            - Get the hvac system total exhaust fan kW, need to get the third element in the list of fan powers:`hvac_sys_total_exhaust_fan_power_b = fan_sys_kW[2]` 
+            - Get the hvac system total relief fan kW, need to get the fourth element in the list of fan powers:`hvac_sys_total_relief_fan_power_b = fan_sys_kW[3]`     
+
+            - Calculate the total modeled fan power for the hvac system in the baseline: `total_fan_power = hvac_sys_total_supply_fan_power_b + hvac_sys_total_return_fan_power_b + hvac_sys_total_exhaust_fan_power_b + hvac_sys_total_relief_fan_power_b`  
+
+            - Get the total supply cfm, need to get the first element in the list of fan CFMs: `supply_cfm_b = fan_sys_CFM[0]`  
+            - Cycle through each zone associated with the HVAC system to get the exhaust fan power: `for zone_b in zone_list:`  
+                - Create a zone object from the zone id (not sure how the RCT will do this): `zone_b_obj = zone_b`  
+                - Check if there are any exhaust fan objects associated with the zone: `if zone_b_obj.zonal_exhaust_fan != Null:`  
+                    - Get the fan power: `fan_elec_power = get_fan_object_electric_power(B_RMI,zone_b_obj.zonal_exhaust_fan)`    
+                    - Add to the total power: `total_fan_power = total_fan_power + fan_elec_power`     
+                - Get the analogous zone object in the P_RMI: `zone_p = match_data_element(P_RMR,Zone,zone_b)` 
+                - Check if the zone has mechanical cooling in the proposed: `If (zone_p.non_mechanical_cooling_fan_airflow != Null AND zone_p.non_mechanical_cooling_fan_airflow != 0): zones_served_by_BL_hvac_has_non_mech_cooling = True`  
+                - Get list of hvac systems serving this zone in the proposed design model: `hvac_sys_list_p = Get_list_hvac_systems_associated_with_zone(P_RMI, zone_p)`  
+                - Check if any of the hvac systems serving the zone have non-mechanical cooling as the cooling_system type: `for hvac_p in hvac_sys_list_P:`  
+                    - Check if the system has non-mechanical cooling: `if hvac_p.cooling_system.cooling_system_options == "NON_MECHANICAL": zone_hvac_in_PR_has_non_mech_cooling = True`  
+
+            - Calculate W/CFM: `fan_power_W_CFM = total_fan_power/supply_cfm_b`                 
+            
+            - **Rule Assertion:** 
+            - Case 1: If fan_power_W_CFM equals 0.3: `if zone_hvac_in_PR_has_non_mech_cooling == False and zones_served_by_BL_hvac_has_non_mech_cooling == False and fan_power_W_CFM == 0.3: outcome = "PASS"`  
+            - Case 2: Elif conservative comparison less equal:`elif AHJ_RA_compare == True and fan_power_W_CFM <= 0.3: PASS`  
```

Comment:
Seems like Case 2 and Case 3 should be swapped, because if project has non-mechanical cooling and fan power is less than 0.3 then it is even more conservative than in the case of no non-mechanical cooling.

Reply:
@mkarpman I'm not following, if the fan power is less than 0.3 then it will pass with case 2 regardless of mech cooling so this seems like the right order, no?

Reply:
on a separate note, i'm not sure if the conservative outcome is correct. System 9 has a fossil fuel furnace for heating. With lower fan power, there will be higher heating energy consumption through the furnace which will have a lower efficiency than an electric fan where 100% of the fan heat is added to the air stream..

Reply:
Remove this per discussion on 2/16/2023.